specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimdeployment.json

```diff
@@ -623,6 +623,42 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ApiManagement/getDomainOwnershipIdentifier": {
+      "post": {
```

Comment:
>"post" [](start = 6, length = 6)

If this is not returning a secret (per the comment below), what's the motivation to model this a POST?

Reply:
This is not a resource, but subscription wide action to get a token that can be used to prove domain name ownership via TXT record.

Reply:
Makes sense!